src/domUpdates.js

```diff
@@ -56,15 +62,33 @@ function renderFilteredRecipes() {
       </div>`)
 };
 
+function viewSelectedRecipe(event){
+  const recipeName = event.target.id;
+  const selectedRecipe = findRecipe(recipeData, recipeName);
+  const recipeCost = calculateRecipeCost(selectedRecipe, ingredientsData);
+  const ingredients = findIngredientNames(recipeData, recipeName);
+  const instructions = recipeInstructions(selectedRecipe);
+  addHiddenClass([allFilterDisplay]);
+  singleRecipeDisplay.innerHTML= `
+  <h2>${selectedRecipe.name}</h2>
+  <img id="${selectedRecipe.id}" src="${selectedRecipe.image}" class="recipe" alt='${selectedRecipe.name}'>
+  <p class="total-cost-box">This recipe costs a total of: $${recipeCost} to make!</p>
+  <p class="ingredient-box">The ingredients you will need to make this recipe are: <br>
+  ${ingredients}</p>
+  <p class="instruction-box">Instructions: <br>
+  ${instructions}</p>`;
+}
+
```

Comment:
I love how this function is formatted! I like how you assigned variables to each invocation.